spec/db_query_matchers/make_database_queries_spec.rb

```diff
@@ -256,6 +256,39 @@
       end
     end
 
+    context 'with ActiveRecord cache' do
+      let(:pattern) { /SELECT.*FROM.*cats/ }
+      subject do
+        ActiveRecord::Base.connection.cache do
+          2.times { Cat.first }
+        end
+      end
+
+      context 'when a `ignore_cached` option is true' do
+        before do
```

Comment:
You probably want an `after {}` to reset `ignored_cached` back to `false`.
Don't want a change in this value to affect other specs.

NOTE: I do not have merge rights - so don't feel you need to make these changes